CoreGraphFile/src/au/gov/asd/tac/constellation/graph/file/GraphPluginRegistry.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2010-2019 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.graph.file;
+
+import au.gov.asd.tac.constellation.graph.file.open.OpenFilePlugin;
+
+/**
+ * List of all core actions/plugins
+ *
+ * @author canis_majoris
+ */
+public final class GraphPluginRegistry {
```

Comment:
While there isn't a `GraphPluginRegistry` yet, that name should probably be reserved for the Graph Framework module and this class should probably be `GraphFilePluginRegistry`